subtrack.MAUI/Services/SubscriptionsCalculator.cs

```diff
@@ -10,4 +10,13 @@ public static decimal GetMonthlyCost(this IEnumerable<Subscription> subscription
         var price = subscriptions.Sum(x => x.Cost);
         return price;
     }
+
+    public static int GetDueDays(this Subscription subscription)
+    {
+        if (subscription is null) return 0;
```

Comment:
More appropiate to throw a exception here - throw new ArgumentNullException(nameof(subscription));